src/tracks/InstrumentTrack.cpp

```diff
@@ -393,6 +393,14 @@ void InstrumentTrack::processOutEvent( const MidiEvent& event, const MidiTime& t
 	const MidiEvent transposedEvent = applyMasterKey( event );
 	const int key = transposedEvent.key();
 
+	// If we have a selected output midi channel between 1-16, we will use that channel to handle the midi event.
+	// But if our selected midi output channel is 0 ("--"), we will use the event channel instead.
+	auto handleEventOutputChannel = midiPort()->realOutputChannel();
+	if( midiPort()->outputChannel() == 0 )
+	{
+		handleEventOutputChannel = event.channel();
+	}
```

Comment:
Can you use a ternary operator here instead? I think it would be easier to read, and it means you can make `handleEventOutputChannel` const.

Reply:
Sure! I agree, I think it improved readability. Just rebasing my repo and I'll update the PR!